Frontend/Models/Forms/AdditionalInformationViewModel.cs

```diff
@@ -1,9 +1,13 @@
-﻿namespace Frontend.Models.Forms
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace Frontend.Models.Forms
 {
     public class AdditionalInformationViewModel
     {
         public string Urn { get; set; }
         public bool AddOrEditAdditionalInformation { get; set; }
+        
+        [BindProperty]
```

Comment:
Not sure if we need this as the whole view model is bound in pupilnumbers razor page, still appears to work locally without it

Reply:
good spot, forgot about that